client/source/components/questionsAnswers/LoadAddQA.jsx

```diff
@@ -1,9 +1,41 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { LoadContainer } from './styles/Containers.styled';
+import ProdContext from '../../ProdContext.js';
+import QuestionForm from './QuestionForm';
+import Modal from '../shared/Modal';
+import useModal from '../../useModal';
+import ModalFormTitle from './ModalFormTitle';
+import * as requests from '../../utilities/axiosRequests.js';
 
 const LoadAddQA = function CreateLoadAddQAComponent({
   questionCount, setQuestionCount
 }) {
+  const { isOpen, onOpen, onClose } = useModal();
+  const { prodID, product } = useContext(ProdContext);
+
+  const postQuestion = (data) => {
+    const obj = {
+      body: data.question, name: data.name, email: data.email, product_id: prodID
+    };
+    return requests
+      .post('/qa/questions', obj);
+  };
+
+  const handlePost = (event, input) => {
```

Comment:
Nice job on the post! I saw a successful 201 when I input all the required fields. 

There is some interesting behavior when I don't fill out a required field, I get the prompt, and upon closing the prompt, it still submits the form when I haven't added anything to the field that is a requirement.

Reply:
Thank you for pointing this out and thank you for the kind words! I also saw the same post request going through despite not having all required inputs filled in. I changed the logic to account for this, so we should be good now on this.